qulice-maven-plugin/src/main/java/com/qulice/maven/PomXpathValidator.java

```diff
@@ -39,11 +39,14 @@
 import org.apache.commons.io.FileUtils;
 
 /**
- * Check pom.xml with XPath validation queries.
+ * <p>Check pom.xml with XPath validation queries.</p>
  *
- * Restrictions:
- * 1. Each xpath component should contains namespace prefix pom:
- * 2. Each xpath query should end with /text()
+ * <p>Restrictions:</p>
```

Comment:
@ooxi generally we do only start of paragraph (see e.g. StringLiteralsConcatenationCheck.java), please remove the `</p>`